### PR TITLE
[Docs] Refer to individual plugins on doc landing page

### PR DESCRIPTION
**Pull request by robbavey**

### CODE REVIEW

**Comment by robbavey** [comment]
@karenzone - Would you mind looking at the docs for the JDBC integration plugin? This includes this PR to create a landing page, and the general structure of the docs pages underneath, and make sure that it conforms

File: docs/index.asciidoc [hunk 1]
```diff
@@ -21,6 +21,10 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-The JDBC Integration Plugin provides integrated plugins for working with Databases.
+The JDBC Integration Plugin provides the following plugins for working with databases that provide a JDBC interface:
```
Comment (karenzone) [inline docs/index.asciidoc]: I'd like to standardize on the wording used for integrations.  Something like:

"The JDBC Integration Plugin provides integrated plugins for working with https://www.oracle.com/technetwork/java/javase/jdbc/index.html[JDBC]."

Comment (karenzone) [reply docs/index.asciidoc]: If you decide to include a link, please verify that the sample link I provided is the best resource before using it.

Comment (robbavey) [reply docs/index.asciidoc]: @karenzone Is it OK to not include a link? I'm not sure it adds a huge amount of value - it mentions a few different Java database technologies, with a dead link to a JDBC overview...  

Does something like this work?

"The JDBC Integration Plugin provides integrated plugins for working with Databases that provide JDBC drivers?"

Comment (karenzone) [reply docs/index.asciidoc]: Absolutely the right call.  If we can't identify a definitive link, then let's skip it. What you are suggesting is fine, which one minor tweak. "Databases" should be lowercase.

**Comment by robbavey** [comment]
@karenzone I pushed the change you requested. 

Also, is the rest of the doc structure in this plugin ok?

**Comment by robbavey** [comment]
Thank you @karenzone!

**Pull request merged**